tools/triage/create_triage_tickets.py

```diff
@@ -53,18 +56,29 @@ def main(args):
         pull_secret_tmp_file.write(args.pull_secret_contents)
         pull_secret_tmp_file.flush()
 
-    res = requests.get("{}/files/".format(LOGS_COLLECTOR))
-    res.raise_for_status()
-    failed_clusters = res.json()
+    signatures_source = SignaturesSource(args.signatures_source)
+
+    if signatures_source == SignaturesSource.JIRA:
+        failed_clusters = jira_client.search_issues(
+            f"project = {JIRA_PROJECT} AND created >= -{DEFAULT_DAYS_TO_HANDLE}d"
+        )
+
+    else:
+        res = requests.get("{}/files/".format(LOGS_COLLECTOR))
+        res.raise_for_status()
+        failed_clusters = res.json()
 
     for failure in failed_clusters:
-        date = failure["name"].split("_")[0]
+        failure_name = (
+            failure.fields.summary.split()[2] if signatures_source == SignaturesSource.JIRA else failure["name"]
+        )
+        date = failure_name.split("_")[0]
         days_ago_creation = days_ago(date)
 
         if not args.all and days_ago_creation > DEFAULT_DAYS_TO_HANDLE:
             continue
 
-        logs_url = get_cluster_logs_base_url(failure["name"])
+        logs_url = get_cluster_logs_base_url(failure_name)
```

Comment:
Why do we still need this? wouldn't this be unnecessary when JIRA signature source it's in use?

Reply:
@osherdp and I discussed it and figured it would be a good idea having both capabilities

Reply:
I meant having links pointing to the web server even when the file is coming from JIRA.
I agree that 2 modes are useful, but shouldn't we link the issue's uploaded file (or maybe nothing) instead of the nginx when the data comes from jira attachment?

Reply:
Oh got you. I missed that
fixing now thanks